temba/orgs/views.py

```diff
@@ -995,13 +995,13 @@ class CreateLogin(SmartUpdateView):
 
         def pre_process(self, request, *args, **kwargs):
             secret = self.kwargs.get('secret')
-            
+
             org = self.get_object()
             if not org:
                 messages.info(request, _("Your invitation link is invalid. Please contact your organization administrator."))
                 return HttpResponseRedirect(reverse('public.public_index'))
             return None
-        
+
         def pre_save(self, obj):
```

Comment:
What editor are you using? looks like it is replacing these with tabs or something?